src/common/snippets/tests/src/vector_scalar.cpp

```diff
@@ -19,56 +19,81 @@ using namespace ngraph;
 
```

Comment:
Renamed.
As for second point, I see that it's legacy test. Can we rewrite these tests in another time? :)
Because in this PR it's not important thing as I believe

Reply:
I generally agree, it's not a top priority for us right now.